css/components/inputs/radio.css

```diff
@@ -0,0 +1,48 @@
+.radio-container {
+  display: block;
+  position: relative;
+  padding-left: 2rem;
+  cursor: pointer;
+}
+
+.radio-container .form-control[type="radio"] {
+  position: absolute;
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.radio-container .radio-custom {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  background-color: var(--clr-secondary-light);
+  border-radius: 50%;
+  transition: 0.3s background-color;
+}
```

Comment:
Consider keeping radio and checkbox styles together as there are multiple lines of code that are similar in both.

Reply:
Thanks for your review! Code is optimized now.